### ISSUE TITLE
INSPIRE namespace element is not present in SpatialDatasetIdentifier

### ISSUE THREAD

**Issue by matthesrieke** [52North/SOS#230]
This is the current output of an INSPIRE SDSI:

```xml
<inspire_dls:SpatialDataSetIdentifier>
   <inspire_common:Code>http://www.52north.org/test/offering/1</inspire_common:Code>
</inspire_dls:SpatialDataSetIdentifier>
```
```<inspire_common:Namespace>``` is missing here.

Probably, see:
https://github.com/52North/SOS/blob/e0eeee8cb5df425908b6d13df02fcec30d24d3d7/extensions/inspire/code/src/main/java/org/n52/sos/inspire/capabilities/InspireExtendedCapabilitiesProvider.java#L271

**Comment by CarstenHollmann** [comment]
But the ![XML schema](http://inspire.ec.europa.eu/schemas/common/1.0/common.xsd) defines the Namespace as *optional*

```
...
<xs:complexType name="uniqueResourceIdentifier">
   <xs:sequence>
      <xs:element name="Code" type="notEmptyString"/>
      <xs:element name="Namespace" type="xs:anyURI" minOccurs="0"/>
   </xs:sequence>
   <xs:attribute name="metadataURL" type="xs:anyURI"/>
</xs:complexType>
...
```

A solution for this would be to add a Namespace setting to the SOS which can be defined in the settings of the admin interface.

**Comment by CarstenHollmann** [comment]
Fixed in pull request #235
